src/kernel/include/fpu.h

```diff
@@ -105,12 +104,12 @@ fpu_init(void)
 		return -1;
 	}
 #endif
-
 	fpu_set(FPU_DISABLE);
+	printk("\n\n\n FPU_DISABLED");
```

Comment:
Print outs are good for testing, but remember to remove them after testing since they'll cause significant overheads.